lib/http/apps/coin.js

```diff
@@ -31,4 +32,41 @@ app.post(`/mint`, (req, res) => {
     });
 });
 
+app.post(`/transfer`, (req, res) => {
+    let to = req.body.to;
+    let amount = req.body.amount;
+
+    res.locals.coinsence.Coin.functions.transfer(to, amount, { gasLimit: 5000000 }).then(transaction => {
```

Comment:
we should not add that `{ gasLimit: 5000000 }` as this is only needed locally afaik.